gradle/dependencies.gradle

```diff
@@ -25,7 +25,11 @@ subprojects {
     compileOnly("com.github.bsideup.jabel:jabel-javac-plugin:${jabelVersion}")
     testCompileOnly("com.github.bsideup.jabel:jabel-javac-plugin:${jabelVersion}")
 
-    api("org.seleniumhq.selenium:selenium-java:$seleniumVersion")
+    api("org.seleniumhq.selenium:selenium-java:$seleniumVersion") {
+      exclude group: 'io.opentelemetry'
+      exclude group: 'org.slf4j'
+      exclude group: 'net.bytebuddy'
```

Comment:
don't do that because https://github.com/selenide/selenide/issues/1850

Reply:
wow. 
Oops.
We should add tests to avoid this in future.